openflow13/nx_match.go

```diff
@@ -267,3 +267,67 @@ func NewConjIDMatchField(conjID uint32) *MatchField {
 
 	return field
 }
+
+func NewNxARPShaMatchField(addr net.HardwareAddr, mask net.HardwareAddr) *MatchField {
```

Comment:
I believe that this and the following function can be refactored in terms of a common helper.